client/biome.json

```
@@ -1,44 +1,72 @@
-			".astro",
-			".cache",
-			".git",
-			".pnpm",
-			".yarn",
-			"dist",
-			"out",
-			"public",
-			".vscode",
-			"src/styles/global.css",
-			"worker-configuration.d.ts"
+		"includes": [
```

Biome (1.9.4)

[error] Found an unknown key `includes`.

Known keys:


(deserialize)